### ISSUE TITLE
[MOD: Plasma Alpha/Beta] Зависание Ксаша при попытке выйти из игры с этим модом.

### ISSUE THREAD

**Issue by unnamed9** [FWGS/xash3d-fwgs#1147]
При попытке выйти из игры, когда запущен мод Plasma Alpha/Beta, Ксаш намертво зависает.
@Vladislav4KZ так-же говорил в Discord'е, что такая проблема есть ещё у мода Poke646.

**Comment by Vladislav4KZ** [comment]
> Vladislav4KZ так-же говорил в Discord'е, что такая проблема есть ещё у мода Poke646.

Да, потому-что я про это уже писал здесь: https://github.com/FWGS/xash3d-fwgs/issues/545

**Comment by Vladislav4KZ** [comment]
Исправлено в: https://github.com/FWGS/xash3d-fwgs/commit/687c9b004f4477d7e325bf138e16c097f0b60f85
